string arucoId
---
geometry_msgs/Pose seatLocation
string guestName 
